Framework/Core/include/Framework/AnalysisDataModel.h

```diff
@@ -137,13 +137,27 @@ namespace collision
 {
 DECLARE_SOA_COLUMN(TimeframeID, timeframeID, uint64_t, "fID4Timeframes");
 DECLARE_SOA_COLUMN(NumTracks, numTracks, uint32_t, "numTracks");
-DECLARE_SOA_COLUMN(NumCalo, numCalo, uint32_t, "numCalo");
-DECLARE_SOA_COLUMN(NumMuons, numMuons, uint32_t, "numMuons");
+//DECLARE_SOA_COLUMN(NumCalo, numCalo, uint32_t, "numCalo");         // not in the preliminary table
+//DECLARE_SOA_COLUMN(NumMuons, numMuons, uint32_t, "numMuons");      // not in the preliminary table
+DECLARE_SOA_COLUMN(ID, identifier, int, "identifier");               // index of the vertex inside the timeframe, AliESDVertex, ushort_t
+DECLARE_SOA_COLUMN(PositionX, positionX, double, "positionX");       // vertex x position, AliVertex.h, Double32_t
+DECLARE_SOA_COLUMN(PositionY, positionY, double, "positionY");       // vertex y position, AliVertex.h, Double32_t
+DECLARE_SOA_COLUMN(PositionZ, positionZ, double, "positionZ");       // vertex z position, AliVertex.h, Double32_t
+//DECLARE_SOA_COLUMN(Cov[3][3], fCov[3][3], double, "fCov");         // 3x3 vertex covariance matrix
+DECLARE_SOA_COLUMN(Chi2, chi2, double, "chi2");                      // chi2 of vertex fit, AliESDVertex.h, Double32_t
+//DECLARE_SOA_COLUMN(Indices, fIndices, int*, "indices");            // contributing track IDs to the vertex reconstruction, AliVertex.h, UShort_t*
+DECLARE_SOA_COLUMN(BC, bunchCrossNumber, int, "bunchCrossNumber");   // LHC bunch crossing number, AliESDHeader.h, UShort_t
+DECLARE_SOA_COLUMN(OrbitNumber, orbitNumber, int, "orbitNumber");    // LHC orbit number, AliESDHeader, UInt_t
+DECLARE_SOA_COLUMN(PeriodNumber, periodNumber, int, "periodNumber"); // period number, AliESDHeader, UInt_t
+DECLARE_SOA_COLUMN(V0mult, v0mult, int, "v0mult");                   // V0 multiplicity
+DECLARE_SOA_COLUMN(T0multA, t0multA, int, "t0multA");                // T0 A multiplicity
+DECLARE_SOA_COLUMN(T0multC, t0multc, int, "t0multC");                // T0 C multiplicity
+DECLARE_SOA_COLUMN(FITmult, fitmult, int, "fitmult");                // FIT  multiplicity
```

Comment:
I think the multiplicities should not belong to the collision, they can be calculated on the fly using the corresponding detector information.

Reply:
Hi @pzhristov , discussing with @naghmehmohammadi I told that the collision will have associated mult. and time info from the FIT. I still don't know how much info (per channel or summary) from FIT detectors we will store in the AOD. If detailed info should be stored, then indeed one can calculate mult. on the fly. Also, we may eventually separate FIT info to the different table since we can have much more FIT events than the vertices.

Reply:
Hi @shahor02, for the moment we can store the amplitudes and the times of each channel as float in two separate tables (FITV0 and FITT0) as in the detector itself. Concerning the number of FIT events, what is the expected rate?

Reply:
l don't know the rate since the FIT is sensitive to the QED, this should be studied with proper simulation. In PbPb it will be certainly higher than the rate of events with vertex.